### PR TITLE
Remove algebird serializers

### PR DESCRIPTION
**Pull request by adamkozuch**
Issue #861
I added chill-algebird to build.scala and removed AlgebirdSerializers.scala from scalding.

### CODE REVIEW

File: scalding-core/src/main/scala/com/twitter/scalding/serialization/KryoHadoop.scala [hunk 1]
```diff
@@ -61,7 +64,7 @@ class KryoHadoop(config: Config) extends KryoInstantiator {
     newK.register(classOf[DateRange], new DateRangeSerializer())
     newK.register(classOf[Args], new ArgsSerializer)
     // Some of the monoids from Algebird that we use:
-    newK.register(classOf[com.twitter.algebird.AveragedValue], new AveragedValueSerializer)
+    newK.register(classOf[com.twitter.algebird.AveragedValue], new  AveragedValueSerializer)
```
Comment (johnynek) [inline scalding-core/src/main/scala/com/twitter/scalding/serialization/KryoHadoop.scala]: Looks like we are committing an extra space. Let's remove that.

**Comment by johnynek** [comment]
Nice. Thanks for doing this.

**Comment by adamkozuch** [comment]
Is it ok now ?

File: scalding-core/src/main/scala/com/twitter/scalding/serialization/KryoHadoop.scala (hunk 1 above)
Comment (johnynek) [inline scalding-core/src/main/scala/com/twitter/scalding/serialization/KryoHadoop.scala]: I'll commit it, but see in line 64, where before it said: "new AveragedValuSerializer" now it has two spaces: "new  AveragedValueSerializer".

The larger point is that your diff should be as minimal as possible when making a functional change (such as this one). Style changes with zero functional changes can be larger (as review is simpler: it is only making sure all the changes are stylistic).

**Comment by johnynek** [comment]
Close enough. :) Thanks for helping!

File: scalding-core/src/main/scala/com/twitter/scalding/serialization/KryoHadoop.scala (hunk 1 above)
Comment (adamkozuch) [inline scalding-core/src/main/scala/com/twitter/scalding/serialization/KryoHadoop.scala]: Sorry. I didn't notice space. Thanks for advice. I will remember it.

**Pull request merged**